### PR TITLE
chore(client): finishing translations cleanup

### PR DESCRIPTION
**Pull request by Dracks**

### CODE REVIEW

File: client/assets/translations/base.pot [hunk 1]
```diff
@@ -102,23 +89,10 @@ msgstr ""
...
-#: levels/game.tscn
-msgid "2\n"
-msgstr ""
-
 #: levels/game.tscn
 msgid "NEXT_ROUND"
 msgstr ""
 
-#: levels/game.tscn
-msgid "Player 1 Turn"
```
Comment (Dracks) [inline client/assets/translations/base.pot:119]: I'm not sure if this one should be dropped or transformed into a msgId. 

Also, maybe makes sense to have something like "Player {{playerNum}} Turn" that can be translated in spanish like "Turno del juegador {{playerNum}}"

Comment (HunterNN) [reply client/assets/translations/base.pot:119]: Agree. Have you tested it in the game? Does the turn announcement still work?

Comment (Dracks) [reply client/assets/translations/base.pot:119]: this is not needed, with the current improvements with {player_name}, it shows like this:
image

File: client/logic/ui/round_label.gd [hunk 2]
```diff
@@ -4,7 +4,8 @@ extends Label
 @export var player_name: String:
 	set(value):
 		player_name = value
-		text = "%s %s %s" % [tr("PLAYER"), str(value), tr("TURN")]
+		# text = "%s %s %s" % [tr("PLAYER"), str(value), tr("TURN")]
+		text = tr("PLAYER_TURN", "Round announcement, showing which player's turn it is.").format({player_name= value})
```
Comment (HunterNN) [inline client/logic/ui/round_label.gd:8]: I'd drop the context, making it general, and move the removed text to the Weblate comment. The ID could also be changed to `ANNOUNCEMENT_PLAYER_TURN` if you think the current one is too general.

**Review by HunterNN — CHANGES_REQUESTED** [review]

File: client/logic/ui/round_label.gd (hunk 2 above)
Comment (Dracks) [reply client/logic/ui/round_label.gd]: Fix!

**Review by HunterNN — APPROVED** [review]

**Pull request merged**